acvm/src/pwg/oracle.rs

```diff
@@ -0,0 +1,54 @@
+use std::collections::BTreeMap;
+
+use acir::{circuit::opcodes::OracleData, native_types::Witness, FieldElement};
+
+use crate::{OpcodeNotSolvable, OpcodeResolution, OpcodeResolutionError};
+
+use super::{arithmetic::ArithmeticSolver, directives::insert_witness};
+
+pub struct OracleSolver;
+
+#[allow(clippy::enum_variant_names)]
+pub enum GateStatus {
```

Comment:
I don't think we need this, do we?
I added it from a copy/paste of the arithmetic solver, but it was more accidental than intentional :)

Reply:
I am sorry, it was indeed unclear, but I meant to remove the GateStatus enum.

Reply:
Removed